students/alexander_boone/lesson06/assignments/test_mailroom.py

```diff
@@ -0,0 +1,96 @@
+#!/usr/bin/env python
+
+import mailroom as mail
+import os
+
+# ----- TEST FUNCTIONS -----
+
+
+def get_updated_donors_list():
+    '''Generate updated donors variables for testing purposes.'''
+    return mail.donors.keys()
+
+
+def get_text(filepath):
+    '''Get text from the specified written letter for testing.'''
+
+    with open(filepath, "r") as readfile:
+        contents = readfile.read()
+        return contents
+
+# ----- TEST CODE -----
+
+
+def test_add_donor():
+    """Test the add_donor function."""
+
+    mail.add_donor('Alexander Boone')
+    donors_list = get_updated_donors_list()
+    assert any(donor == 'Alexander Boone' for donor in donors_list)
```

Comment:
donors_list will contain a unique list of donors - why not just `'Alexander Boone' in donors_list`